classroom/snippets/manage_aliases.py

```diff
@@ -0,0 +1,43 @@
+from __future__ import print_function
+from googleapiclient.discovery import build
+from httplib2 import Http
+from oauth2client import file, client, tools
+
+# If modifying these scopes, delete the file token.json.
+SCOPES = 'https://www.googleapis.com/auth/classroom.courses'
+
+def main():
```

Comment:
This is a copy of the quickstart, which is not how we structure snippets. Take a look at the Sheets snippets for example: https://github.com/gsuitedevs/python-samples/blob/master/sheets/snippets/spreadsheet_snippets.py